packages/addons/README.md

```diff
@@ -0,0 +1,49 @@
+#Addon skeleton
+This is an example how the default structure of an addon should look like.
+Please be aware that this is an moving target and could be changed to adapt changes at the build system.
+
+Lines labelled with `#` are completely optionally (beside the license) and should be omitted if you don't need them.
```

Comment:
Except for the license header, any lines prefixed with `#` are optional and may be omitted if not required.